bin/mk-dev-cert

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
```

Comment:
Little script to generate new development SSL certs using `mkcert` behind the scenes.  Notably ensures that we're always using the same rootCA for sanity purposes.